src/useInterval/index.ts

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useRef } from "react"
+
+// Ref: https://github.com/Hermanya/use-interval/blob/master/src/index.tsx
+
+// tslint:disable-next-line
+const noop = () => {}
```

Comment:
```suggestion
const noop = () => undefined
```

I guess this will work without `tslint:disable-next-line`

Reply:
We can also import `noop` from `_`.

Reply:
`const noop = () => undefined` will throw type errors, will import noop from lodash